docker/docker_start.sh

```diff
@@ -133,7 +133,7 @@ if [ "$DISABLE_POSTOVERFLOWS" != "" ]; then
 fi
 
 function register_bouncer {
-  if ! cscli -c "$CS_CONFIG_FILE" bouncers list -o json | jq -r .[].name | grep -q "${NAME}"; then
+  if ! cscli -c "$CS_CONFIG_FILE" bouncers list -o raw | cut -d "," -f1 | grep -q "${NAME}"; then
```

Comment:
If you want to support names with comma or space:
cscli bouncers list -o json | sed '/^ *"name"/!d;s/^ *"name": "\(.*\)",/\1/'

then apply grep -q
tested with busybox too